.github/workflows/go.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       id: go
 
     - name: Check out code into the Go module directory
-      uses: actions/checkout@v1
+      uses: actions/checkout@v2
```

Comment:
??? is this related to api version? 😄

Reply:
It is not, but it's necessary to make the build pass, unfortunately it seems that there's some bug which was fixed in v2+ and now the checkout v1 versions fail. (see the other open PR too)